lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -378,11 +387,28 @@ ifeq (1, $(USE_SYSTEM_STDLIB))
 	endif
 endif
 
+# No C++ library has been specifieed. Use libstdc++ by default.
+ifeq (,$(filter 1, $(USE_LIBSTDCPP) $(USE_LIBCPP) $(USE_SYSTEM_STDLIB)))
+  # If no explicit request was made, but we have paths to a custom libcxx, use
+  # them.
+  ifneq ($(and $(LIBCPP_INCLUDE_DIR), $(LIBCPP_LIBRARY_DIR)),)
+    CXXFLAGS += -nostdlib++ -nostdinc++ -cxx-isystem $(LIBCPP_INCLUDE_DIR)
+    ifneq "$(LIBCPP_INCLUDE_TARGET_DIR)" ""
+      CXXFLAGS += -cxx-isystem $(LIBCPP_INCLUDE_TARGET_DIR)
+    endif
+    LDFLAGS += -L$(LIBCPP_LIBRARY_DIR) -Wl,-rpath,$(LIBCPP_LIBRARY_DIR) -lc++
+  # Otherwise no C++ library has been specified. Use stdc++ by default.
+  else
+    USE_SYSTEM_STDLIB := 1
+  endif
+endif
```

Comment:
```suggestion
ifeq (,$(filter 1, $(USE_LIBSTDCPP) $(USE_LIBCPP) $(USE_SYSTEM_STDLIB)))
  # If no explicit C++ library request was made, but we have paths to a custom libcxx, use
  # them.  Otherwise, use the system library by default.
  ifneq ($(and $(LIBCPP_INCLUDE_DIR), $(LIBCPP_LIBRARY_DIR)),)
    CXXFLAGS += -nostdlib++ -nostdinc++ -cxx-isystem $(LIBCPP_INCLUDE_DIR)
    ifneq "$(LIBCPP_INCLUDE_TARGET_DIR)" ""
      CXXFLAGS += -cxx-isystem $(LIBCPP_INCLUDE_TARGET_DIR)
    endif
    LDFLAGS += -L$(LIBCPP_LIBRARY_DIR) -Wl,-rpath,$(LIBCPP_LIBRARY_DIR) -lc++
  else
    USE_SYSTEM_STDLIB := 1
  endif
endif
```

(I'm just fixing the libstdc++ reference in the comments and moving them around)

Reply:
Thanks!